policy-engine/src/main.rs

```diff
@@ -129,6 +131,18 @@ fn main() -> Result<(), Error> {
     Ok(())
 }
 
+// log errors in case an incorrect endpoint is called
+fn default_response(req: HttpRequest) -> HttpResponse {
+    error!(
+        "Error serving request '{}' from '{}': Incorrect Endpoint",
+        format!("{:?}", req).replace("\n", " ").replace("\t", " "),
+        req.peer_addr()
```

Comment:
I dunno if we care about [`peer_addr`][1], since that will almost always be the router proxy, right?  I expect we want to include the forwarder chain, or assume the forwarder chain doesn't contain spoofed input and use [`realip_remote_addr`][2]?

[1]: https://docs.rs/actix-web/3.3.2/actix_web/struct.HttpRequest.html#method.peer_addr
[2]: https://docs.rs/actix-web/3.3.2/actix_web/dev/struct.ConnectionInfo.html#method.realip_remote_addr

Reply:
Never mind, I see `peer_addr` is being recycled from #186.  If we want to adjust that, we can punt to follow-up work.